core/client/fs/src/main/java/alluxio/client/block/policy/BlockLocationPolicy.java

```diff
@@ -68,4 +70,14 @@ public static BlockLocationPolicy create(Class<?> blockLocationPolicyClass,
    */
   @Nullable
   WorkerNetAddress getWorker(GetWorkerOptions options);
+
+  /**
+   * Gets the preferred workers.
+   * @param options the options to get preferred workers
+   * @param count
+   * @return the preferred workers
+   */
+  default Set<BlockWorkerInfo> getPreferredWorkers(GetWorkerOptions options, int count) {
```

Comment:
We can probably just name this `getWorkers` to be consistent with `getWorker`. "preferred" doesn't really add much meaning.

Reply:
Yep, renamed